docs/README.md

```diff
@@ -1,47 +1,43 @@
-1. **-a**: Personal access token to access packages in [VS feed](https://pkgs.dev.azure.com/devdiv/_packaging/VS-CoreXtFeeds/nuget/v3/index.json) [**optional**]. If not specified, props files will not be updated.
-    > _-a:vv8ofhtojf7xuhehrFxq9k5zvvxstrqg2dzsedhlu757_
-1. **-w:** Maximum allowed duration in seconds, excluding downloads [**optional**].  Example: 
-    > _-w:60_
-1. **-ds:** Maximum allowed duration in seconds that can be spent downloading nuget packages [**optional**].  Example: 
-    > _-ds:240_
-1. **-c:** Maximum concurrency of default.config version updates [**optional**].  Example:
-    > _-c:10_
-1. **-bf:** Filter string to specify which builds from the manifest will be inserted.
-    
-    A simple build filter can be written as follows:
-    ```
-    -bf:repo=.*core-setup
-    ```
 
+### <a href="#input">Input</a>
+| Switch | Description | Optional| Example |
+| :-- | :-- | :-- |:-- |
+|**-d:** |Path to the default.config.| NO | _-d:c:\default.config_|
+|**-m:** |manifest.json file path or parent directory.  Note: It is also possible to input multiple manifest files. In this case, the files will be processed in order. If the same package is updated from multiple manifest files, version number from the last specified manifest file will be used. To do this, use semicolon (;) to separate the paths as such: `_m:c:\files\manifest.json;c:\just\folder\\;c:\another\fullpath\manifest.json_`|NO | _manifest.json_ example:</br>_-m:c:\files\manifest.json_|
+|**-i:** |Path to ignored packages file.| YES|_-i:c:\files\ignore.txt_|
+|**-idut** |Indicates that packages relevant to the .NET Dev UX team are ignored. If **-i:** is also set, the file specified with that option is used, superceding **-idut**.|YES|> _-idut_|
+|**-wl:** |Path to the whitelist file [**optional**]. If this is specified, only the packages listed in this file will be updated. Each line in the file represents a regex pattern that will potentially match one or more package IDs. This switch will be ignored if the given file is empty.|YES|> _-wl:c:\files\whitelist.txt_|
+|**-p**: |Path to the directory to search for .props files. If left unspecified, all the .props files under src\SetupPackages in local VS repo will be searched.|YES|`_-p:C:\VS\src\SetupPackages_`|
+|**-a**: |Personal access token to access packages in [VS feed](https://pkgs.dev.azure.com/devdiv/_packaging/VS-CoreXtFeeds/nuget/v3/index.json). If not specified, props files will not be updated.|YES|`_-a:vv8ofhtojf7xuhehrFxq9k5zvvxstrqg2dzsedhlu757_`|
+|**-w:** |Maximum allowed duration in seconds, excluding downloads.|YES  |`> _-w:60_`|
+|**-ds:** |Maximum allowed duration in seconds that can be spent downloading nuget packages |YES|`> _-ds:240_`|
+|**-c:** |Maximum concurrency of default.config version updates.|YES|`> _-c:10_`|
+|**-bf:** |Filter string to specify which builds from the manifest will be inserted.| YES|A simple build filter can be written as follows:</br>`-bf:repo=.*core-setup`|

```

Comment:
I would keep just example in example column without extra wording

Reply:
Removed the text as suggested.